packages/client/utils/reactjiPickerRecent.ts

```diff
@@ -0,0 +1,39 @@
+const DEFAULT_ORDER = {
+  heart: 0,
+  tada: 0,
+  smile: 0,
+  rocket: 0,
+  fire: 0,
+  white_check_mark: 0,
+  confused: 0,
+  cry: 0,
+  x: 0
+}
+
+const RECENT_REACTJI_PICKER = 'recentReactjiPicker'
+
+const getRecent = () => {
+  const localStorageRecent = localStorage.getItem(RECENT_REACTJI_PICKER)
+  return localStorageRecent === null ? DEFAULT_ORDER : JSON.parse(localStorageRecent)
+}
+
+const updateUsage = (emojiId: string) => {
+  const frequentlyUsed = getRecent()
+  frequentlyUsed[emojiId] || (frequentlyUsed[emojiId] = 0)
+  frequentlyUsed[emojiId] += 1
+  localStorage.setItem(RECENT_REACTJI_PICKER, JSON.stringify(frequentlyUsed))
+}
+
+const getRecentArray = () => {
+  const frequentlyUsed = getRecent()
+
+  const sortedFrequentlyUsed = Object.keys(frequentlyUsed).map((emoji) => [
+    emoji,
+    frequentlyUsed[emoji] ?? 0
+  ])
+  sortedFrequentlyUsed.sort((a, b) => b[1] - a[1])
+
+  return Object.keys(sortedFrequentlyUsed).map((emoji) => sortedFrequentlyUsed[emoji][0])
```

Comment:
This can just be `sortedFrequentlyUsed.map(([emoji]) => emoji).slice(0, MAX_RECENT_EMOJIS)`. We slice the array so over time the recent list doesn't grow too much. I think starting with `MAX_RECENT_EMOJIS = 9` is reasonable.

Reply:
I thought `MAX_RECENT_EMOJIS` can be 18 to limit frequently used to 2 rows max but 9 is also reasonable to have just 1 row. After number of iteration, the code is very clean now thanks to you 🚀